lib/Doctrine/Common/Comparable.php

```diff
@@ -38,9 +40,9 @@
      * This method should not check for identity using ===, only for semantical equality for example
      * when two different DateTime instances point to the exact same Date + TZ.
      *
-     * @param mixed $other
+     * @param object $other
      *
      * @return int
      */
-    public function compareTo($other);
+    public function compareTo(object $other): int;
```

Comment:
@Ocramius What do you suggest to do? Would it be ok to simply grep signature changes for now? Ithink creating an extended sheet of changes would be contraproductive ATM, since Common 3.0 might change a lot.